components/python/jinja2/Makefile

```diff
@@ -11,11 +11,12 @@
 #
 # Copyright (c) 2014 Alexander Pyhalov.  All rights reserved.
 # Copyright 2015, PALO Richard.
+# Copyright 2017 Gary Mills
 #
 
 include ../../../make-rules/shared-macros.mk
 
-PATH=/usr/bin:/usr/gnu/bin:/usr/sbin
+PATH = $(PATH.ill):/usr/sbin
```

Comment:
I didn't define a symbol for /usr/perl5/bin because that's only one of many binary directories that appear in the PATH of some component Makefiles.  I only did so for the two most common ones, which is all that we need in practice.  My use was to add two directories for hand-built and hand-installed software, when it was not available as packages.